src/main/java/org/broadinstitute/dsde/consent/ontology/datause/services/TextTranslationService.java

```diff
@@ -1,17 +1,21 @@
 package org.broadinstitute.dsde.consent.ontology.datause.services;
 
 import org.broadinstitute.dsde.consent.ontology.model.DataUseSummary;
+import org.broadinstitute.dsde.consent.ontology.model.Recommendation;
 
 import java.io.IOException;
+import java.util.HashMap;
 
 public interface TextTranslationService {
 
-    enum TranslateFor { DATASET, PURPOSE }
+    enum TranslateFor { DATASET, PURPOSE, PARAGRAPH }
```

Comment:
We're using the enum in the resource class that does use this service. The `PARAGRAPH` case is a bit of an outlier, but it really is a kind of translation in the same vein as dataset and purpose translations.

Reply:
It's used in the service implementation class but not in the service interface class, so it could be moved to the implementation class without changing the interface. Or maybe an impl method should be moved up to the interface, such as `translate(DataUse dataUse, TranslateFor translateFor)`? Then it would be used in both.

Reply:
> Or maybe an impl method should be moved up to the interface, such as `translate(DataUse dataUse, TranslateFor translateFor)`? Then it would be used in both.

I like that ... keeps things consistent between the interface and the implementation.